### PR TITLE
Move selected to files qcstatus

### PR DESCRIPTION
**Pull request by cmadjar**
This pull request moves the selected information of the imaging browser into the files_qcstatus table.
Until now, this information was inserted into the parameter_file table and could be lost if need to reinsert the imaging data.

### CODE REVIEW

**Comment by MounaSafiHarab** [comment]
@cmadjar,

while you have an SQL patch for existing databases, I believe the schema should be updated as well when creating from scratch tables files_qcstatus and parameter_files.

**Comment by cmadjar** [comment]
Very good point! I just added it to the Default schema. Thank you!!

**Comment by MounaSafiHarab** [comment]
Just tested this on my VM before and after applying the SQL patch. Info got copied properly into the added column, and further changes to the Image Panel in Imaging browser (namely the Selected field) resulted in the information being saved in the files_qcstatus table.

File: SQL/2016-03-10-MoveSelectedTo_files_qcstatus_table.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/* ALTER files_qcstatus table to add a Selected column */
```
Comment (driusan) [inline SQL/2016-03-10-MoveSelectedTo_files_qcstatus_table.sql]: These should be -- at the start of the line for a comment, not /* ... */. /* is a php comment, -- is an SQL comment.

**Comment by cmadjar** [comment]
This is fixed now. Thank you!

**Comment by christinerogers** [comment]
Does this have any impact on the DQT loading script or Statistics (Imaging subtab)?

**Pull request merged**